### ISSUE TITLE
[Bug]: Features don't work in blade files

### ISSUE THREAD

**Issue by N1ebieski** [laravel/vs-code-extension#78]
### Extension Version

0.1.12

### PHP Binary

Docker

### Operating System

Linux

### What happened?

All features (autocompletion, links, hovers) work in .php files such as controllers, but not in .blade.php files. Is this a bug or does the extension not offer this at this time?

**Comment by joetannenbaum** [comment]
Yes, we are still porting the functionality over to Blade files, this is on the roadmap:

https://github.com/laravel/vs-code-extension?tab=readme-ov-file#on-the-roadmap

**Comment by TitasGailius** [comment]
Thanks for reporting this. We’re clearing out some older issues that may have been fixed by the latest improvements in the extension. Feel free to reopen if you’re still experiencing anything similar.
